tools/topology/topology2/include/components/src_format_s32_to_sxx_convert.conf

```diff
@@ -0,0 +1,346 @@
+#src format array
```

Comment:
May I know the SRC module real capability? I mean does it really support all input format --> output format sample rate converter?